docs/txp.md

```diff
@@ -186,16 +186,22 @@ with unknown type. There are three cases:
    because `verifyAllIsKnown` is `False`.
 3. All inputs have known format (i. e. `TxInUtxo`). In this case
    (which should be the most common one) we can check all inputs'
-   legitimacy, compare input and output sums, check fee, etc. All
-   inputs must exist in UTXO. In the following we describe the case
-   when all inputs have `TxInUtxo` type and we assume that for each
-   input we know corresponding unspent output (if we don't,
-   transaction is invalid).
+   legitimacy, compare input and output sums, check fee, etc.
+   These checks are described in more details below.
+
+In the following (until [tx size check section](#tx-size-check)) we
+consider case (3), when all inputs are `TxInUtxo`. The next step is to
+lookup all inputs in UTXO. If at least one input is not found,
+transaction is invalid. We further assume that for each input we know
+corresponding unspent output.  All inputs must exist in UTXO. In the
```

Comment:
>  All inputs

two spaces before `All`

Reply:
I think it's quite non-essential, because markdown renderer should take care of that. Anyway, I will fix it here.